java/vector/src/test/java/org/apache/arrow/vector/ITTestLargeVector.java

```diff
@@ -131,16 +139,17 @@ private static void testLargeDecimalVector() {
         assertEquals(0, buf.getLong(8));
 
         if ((i + 1) % 10000 == 0) {
-          System.out.println("Successfully read " + (i + 1) + " values");
+          logger.debug("Successfully read {} values", i + 1);
         }
       }
-      System.out.println("Successfully read " + vecLength + " values");
+      logger.info("Successfully read {} values", vecLength);
     }
-    System.out.println("Successfully released the large vector.");
+    logger.info("Successfully released the large vector.");
   }
 
-  private static void testLargeFixedSizeBinaryVector() {
-    System.out.println("Testing large fixed size binary vector.");
+  @Test
+  public void testLargeFixedSizeBinaryVector() {
+    logger.info("Testing large fixed size binary vector.");
```

Comment:
Could you please explain the difference between this and an ordinary test case? I am curious how it managed to pass the restriction of arrow.vector.max_allocation_bytes?

Reply:
Not sure I understand the question. I will answer what I think but please let me know if I am misunderstanding . these tests are run as part of the failsafe plugin (by way of the IT prefix on the class name) as such they are only run when the `integration-test` profile is active.

I see the `arrow.vector.max_allocation_bytes` is set to `Long.MAX_VALUE` by default and the original test didn't have any docs/settings for changing that default. If it is supposed to be set to a specific value for the test we can specify in failsafe args

Reply:
Thanks a lot for your clarification. You mean the class name has a prefix "IT", so it will not be run as an ordinary unit test, but will be run as an integration test? If so, it is cool!

BTW, the default value of `arrow.vector.max_allocation_bytes` should be 1MB by default (pls see https://github.com/apache/arrow/blob/master/java/pom.xml#406), so if the allocation size exceeds this limit, an exception will be thrown.

Reply:
thanks @liyafan82 based on your feedback I have moved the failsafe plugin/profile up to the main pom and have added the same args as surefire